sdg/components.py

```diff
@@ -14,11 +14,32 @@ def git_clone_op(
     repo_url: Optional[str],
     taxonomy_path: str = "/data/taxonomy",
 ):
+    import os
+
+    # Empty parameters for non-tls git operations
+    additional_clone_params = ""
+    additional_config_cmds = ""
+
+    # Add TLS Parameters if CA Cert exists
+    taxonomy_ca_cert_path = os.getenv("TAXONOMY_CA_CERT_PATH")
+    use_tls = (
+        taxonomy_ca_cert_path
+        and os.path.exists(taxonomy_ca_cert_path)
+        and (os.path.getsize(taxonomy_ca_cert_path) > 0)
+    )
+
+    if use_tls:
+        full_ca_path = os.path.abspath(f"{taxonomy_ca_cert_path}")
```

Comment:
Optional:
```suggestion
        full_ca_path = os.path.abspath(taxonomy_ca_cert_path)
```

Reply:
That variable is actually not a string - i believe its an InputParameterChannel or some other Pipelines construct, which abspath doesnt accept as a string-like type.  Because of that, we need to expand it, as casting this specific type if it is None actually expands to the string "None" which would incorrectly pass this check